public/app/plugins/panel/timeseries/plugins/ExemplarMarker.tsx

```diff
@@ -202,20 +202,23 @@ export const ExemplarMarker = ({
     .getSeries()
     .find((s) => s.props.dataFrameFieldIndex?.frameIndex === dataFrameFieldIndex.frameIndex)?.props.lineColor;
 
+  const onExemplarClick = () => {
+    setClickedExemplarFieldIndex(dataFrameFieldIndex);
+    lockExemplarModal();
+  };
+
   return (
     <>
-      {/* TODO: fix keyboard a11y */}
-      {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
       <div
         ref={setMarkerElement}
-        onClick={() => {
-          setClickedExemplarFieldIndex(dataFrameFieldIndex);
-          lockExemplarModal();
-        }}
+        onClick={onExemplarClick}
+        onKeyDown={onExemplarClick}
```

Comment:
should this be behind a "enter" keypress?

Reply:
yes, my bed